### PR TITLE
Pass correct physics instance to Scrollable semantics

### LINKED ISSUE #24411: [A11Y] Can't use TalkBack swipe gesture to navigate left/right in horizontal ListView to items off screen
In a vertical `ListView`, you can use the left/right swipe gesture to navigate to items off the screen (below or above the viewport).

In a horizontal `ListView`, you are unable to use the left/right swipe gesture to navigate to items off the screen (to the left or right of the viewport).

I tried this on Android. I tried a native Android app to see what would happen and it let me navigate to the items off screen.

### PR DESCRIPTION
**Pull request by jonahwilliams**
Fixes https://github.com/flutter/flutter/issues/24411

Also adds regression test.

### CODE REVIEW

File: packages/flutter/lib/src/material/tabs.dart [hunk 1]
```diff
@@ -993,18 +993,29 @@ class _TabBarState extends State<TabBar> {
...
 
     if (widget.isScrollable) {
+      final ScrollBehavior configuration = ScrollConfiguration.of(context);
+      final ScrollPhysics physics = configuration.getScrollPhysics(context);
       _scrollController ??= _TabBarScrollController(this);
       tabBar = SingleChildScrollView(
         scrollDirection: Axis.horizontal,
         controller: _scrollController,
         child: tabBar,
+        physics: _NoImplicitScrollingPhysics(physics),
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/tabs.dart]: Why are we disabling implicit scrolling for TabBars? It seems like if I have my focus on Tab A and I swipe to move focus to the next element I should land on Tab B even if it is currently offscreen, no?

Comment (jonahwilliams) [reply packages/flutter/lib/src/material/tabs.dart]: I seem to recall the correct behavior for tab bars being no-implicit scrolling. I will verify that this is the case

Comment (jonahwilliams) [reply packages/flutter/lib/src/material/tabs.dart]: I haven't been able to confirm this behavior, so I'll remove this work around for now and we can revist later

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**